ArduPlane/servos.cpp

```diff
@@ -208,6 +208,7 @@ void Plane::flaperon_update()
      */
     float aileron = SRV_Channels::get_output_scaled(SRV_Channel::k_aileron);
     float flap_percent = SRV_Channels::get_slew_limited_output_scaled(SRV_Channel::k_flap_auto);
+    flap_percent = fmin(flap_percent, g.flaperon_limit);
```

Comment:
I'm a big fan of sanitizing your inputs.
```suggestion
    flap_percent = MIN(flap_percent, constrain_float(g.flaperon_limit, 0, 100));
```

Reply:
Here's a never-sacrifice-ailerons solution without a param:
```suggestion
    const float aileron_deflection_percent = fabs(aileron) * (100.0f/4500.0f);
    const float available_percent = 100 - aileron_deflection_percent;
    flap_percent = MIN(flap_percent, available_percent);
```